src/main/java/org/schedulesdirect/api/NetworkEpgClient.java

```diff
@@ -423,6 +422,12 @@ public InputStream submitRequest(final DefaultJsonRequest req, final Object data
 	protected Program fetchProgram(final String progId) throws IOException {
 		return fetchPrograms(new String[] { progId }).values().toArray(new Program[1])[0];
 	}
+	
+	@Override
+	protected Artwork[] fetchArtwork(String progId) throws IOException {
```

Comment:
I will implement.  I didn't need it initially for my needs, but it may be useful to someone in the future.  I have already updated the grabber (locally) to pull the data from SD.